app/client/src/utils/Validators.ts

```diff
@@ -365,6 +365,9 @@ export const VALIDATORS: Record<ValidationType, Validator> = {
     }
     const hasOptions = _.every(parsed, (datum: { label: any; value: any }) => {
       if (_.isObject(datum)) {
+        if(_.isEmpty(datum.label) || _.isEmpty(datum.value)){
+          return false;
+        }
         return _.isString(datum.label) && _.isString(datum.value);
```

Comment:
This is correct but the condition can be more readable as so

```suggestion
        return (
            _.isString(datum.label) 
            && _.isString(datum.value) 
            && !_.isEmpty(datum.label) 
            && !_.isEmpty(datum.value)
        );
```